test/scripts/bridge/cleanup.sh

```diff
@@ -19,9 +19,9 @@ elif [ $1 = "ole" ]; then
   rm -rf $logs_dir
   rm -rf $output_dir
   rm -rf $ethereum_data_dir
-elif [ $1 = "olep" ]; then
+elif [ $1 = "lep" ]; then
   rm -rf $logs_dir
-  rm -rf $output_dir
```

Comment:
we cannot remove the output dir now?

Reply:
I think if anything, I would add the option for that at least.

Reply:
removing the bins is unnessecary step and adds time to subsequent local runs (not to mention breaks caching). Have changed it so we remove everything but the bins

Reply:
the bin versions are determined by constants `set-env.sh`. 
- CI: It will always download the version set here or restore from cache (faster) if it's been downloaded previously 
- Local: It will always use what's already downloaded and print warning if it doesn't match what is set in `set-env.sh`. This allows you to test against different versions locally of the eth nodes